### PR TITLE
feature [nativelib]: partially implement `RuntimeMXBean`

### PR DESCRIPTION
**Pull request by iRevive**
A follow-up to the #4018. 

### Scope

- Extend `scalanative.runtime`: add `uptime` and `startTime`, record process start time
- Partially implement `java.lang.management.RuntimeMXBean`
- Implement `java.lang.management.ManagementFactory.getRuntimeMXBean()`

### CODE REVIEW

File: javalib/src/main/scala/java/lang/management/RuntimeMXBean.scala
Comment (ekrich) [inline javalib/src/main/scala/java/lang/management/RuntimeMXBean.scala:83]: This should use POSIX `{HOST_NAME_MAX}` defined as `_SC_HOST_NAME_MAX` in `unistd.h`. There could be constants, defines, or config items that might not be defined as you go forward.

Comment (iRevive) [reply javalib/src/main/scala/java/lang/management/RuntimeMXBean.scala]: Good point, thanks. I replaced the arbitrary value with `unistd._SC_HOST_NAME_MAX.toCSize.`

**Review by WojciechMazur — APPROVED** [review]

**Review by WojciechMazur — APPROVED** [review]
LGTM, only requires a rebase after merging other changes

**Pull request merged**
